guides/common/modules/proc_reclaiming-postgresql-space.adoc

```diff
@@ -26,3 +26,18 @@ Use this procedure to reclaim some of this disk space on {Project}.
 ----
 # {foreman-maintain} service start
 ----
+. Confirm files are present under `/var/lib/pgsql/` and check its size.
```

Comment:
```suggestion
. Confirm that the files exist in the `/var/lib/pgsql/` directory:
```
This is two steps. Add a separate step for checking the size. One command per step.